packages/core/src/transports/types.ts

```diff
@@ -45,3 +48,12 @@ export interface Transports {
   pause: () => void;
   unpause: () => void;
 }
+
+export interface BatchExecutorOptions {
```

Comment:
Q: in the README i can see a `batchEnabled` property. Is this replaced by `autoStart`?